graphql/tests/repoEvents/data.json

```diff
@@ -0,0 +1,53 @@
+{
+  "repo_ref": [
+    {
+      "_id" : "5678"
+    }
+  ],
+  "project_vars": [
+    {
+    "_id": "5678",
+    "vars": {"hello": "world", "foo":  "bar"},
+    "private_vars": {"hello":  true, "foo":  false}
+    }
+  ],
+  "event_log": [
```

Comment:
It would be helpful to add this data to the repo_ref file in testdata for local development!